### PR TITLE
Change QMCPACK source #include from angle brackets to quotes

### PR DESCRIPTION
**Pull request by ye-luo**
## Proposed changes
1. Change QMCPACK source #include from angle brackets to quotes
2. When XXX.h and XXX.cpp are in the same folder (they should), XXX.cpp should `#include "XXX.h"` without any path.

## What type(s) of changes does this code introduce?
- Code style update (formatting, renaming)

### Does this introduce a breaking change?
- No

## What systems has this change been tested on?
Bora

## Checklist
- Yes. This PR is up to date with current the current state of 'develop'

### CODE REVIEW

**Review by prckent — APPROVED** [review]
LGTM

Hopefully we won't need the script again

File: src/Estimators/CSEnergyEstimator.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 
-#include "Estimators/CSEnergyEstimator.h"
+#include "CSEnergyEstimator.h"
```
Comment (PDoakORNL) [inline src/Estimators/CSEnergyEstimator.cpp:17]: "Estimators/CSEnergyEstimator.h"

File: src/Estimators/TraceManager.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@
 #if !defined(DISABLE_TRACEMANAGER)
 
 
-#include <Estimators/TraceManager.h>
+#include "TraceManager.h"
```
Comment (PDoakORNL) [inline src/Estimators/TraceManager.cpp:17]: Estimators/TraceManager.h

File: src/AFQMC/Drivers/AFQMCDriver.cpp [hunk 3]
```diff
@@ -7,10 +7,10 @@
 #include "OhmmsData/ParameterSet.h"
 #include "OhmmsData/libxmldefs.h"
 #include "Configuration.h"
-#include <Utilities/qmc_common.h>
+#include "Utilities/qmc_common.h"
 
 #include "AFQMC/config.h"
-#include "AFQMC/Drivers/AFQMCDriver.h"
+#include "AFQMCDriver.h"
```
Comment (PDoakORNL) [inline src/AFQMC/Drivers/AFQMCDriver.cpp:13]: AFQMC/Drivers/AFQMCDriver.h

File: src/AFQMC/Drivers/DriverFactory.cpp [hunk 4]
```diff
@@ -7,7 +7,7 @@
 #include "mpi3/communicator.hpp"
 
 #include "AFQMC/Utilities/taskgroup.h"
-#include "AFQMC/Drivers/DriverFactory.h"
+#include "DriverFactory.h"
```
Comment (PDoakORNL) [inline src/AFQMC/Drivers/DriverFactory.cpp:10]: AFQMC/Drivers/DriverFactory.h

**Review by PDoakORNL — CHANGES_REQUESTED** [review]
Maybe a change in the regular expression is needed?

File: src/Estimators/CSEnergyEstimator.cpp (hunk 1 above)
Comment (ye-luo) [reply src/Estimators/CSEnergyEstimator.cpp:17]: This is on purpose.
CSEnergyEstimator.cpp should include CSEnergyEstimator.h without any directory if they are in the same directory.

Comment (PDoakORNL) [reply src/Estimators/CSEnergyEstimator.cpp:17]: https://qmcpack.readthedocs.io/en/develop/developing.html#includes
That's not our published coding standard.
All my new code explicitly qualifies internal include location under src

Comment (PDoakORNL) [reply src/Estimators/CSEnergyEstimator.cpp:17]: I believe "" are looked for in the entire non "system" include path set, this standard is usually thought to remove sloppy dependence on that.

Comment (ye-luo) [reply src/Estimators/CSEnergyEstimator.cpp:17]: https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html
"" looks for the directory of the cpp file first and guarantees matching h an cpp.
I think we need to update the coding standard.

Comment (PDoakORNL) [reply src/Estimators/CSEnergyEstimator.cpp:17]: I am now in agreement especially re goal of cmake targets determining include paths.

**Pull request merged**